src/NemoStuff.jl

```diff
@@ -1,5 +1,7 @@
 export neg!
 
+sub!(z::T, x::T, y::T) where {T} = x - y
```

Comment:
Should there be a similar method for `add!` and/or `mul!`?

Note that in `src/NCRings.jl` we already have this:
```julia
function zero!(z::NCRingElem)
   return zero(parent(z))
end

function add!(z::T, x::T, y::T) where T <: NCRingElem
   return x + y
end

function addeq!(z::T, y::T) where T <: NCRingElem
   return z + y
end

function sub!(z::T, x::T, y::T) where T <: NCRingElem
   return x - y
end

function mul!(z::T, x::T, y::T) where T <: NCRingElem
   return x*y
end

function addmul!(z::T, x::T, y::T, c::T) where T <: NCRingElem
   c = mul!(c, x, y)
   z = addeq!(z, c)
   return z
end

```
So it seems this already does the same thing, the new code here "only" drops the requirement `sub!` (why? probably to support `Int`, `Rational`, etc.?)

Reply:
I noticed that as well, but I would like to leave this PR simply as a copy-paste of stuff. There are a few similar things as well that should be cleaned up in a follow-up PR.

Reply:
Sure.

But just to record my thoughts for the future: I suggest we drop this method here, and move those from `src/NCRings.jl` to `src/fundamental_interface.jl`, but making them generic, i.e. letting `T` be free.... Or indeed, letting all three arguments be "free":  `add!(x,a,b) = a+b` etc.